front/pages/api/v1/data_sources/[user]/[name]/documents/[documentId]/index.ts

```diff
@@ -215,7 +182,7 @@ export default async function handler(req, res) {
       }
 
       // Enforce FreePlan limit: 32 documents per DataSource.
-      if (reqUser.username !== "spolu") {
+      if (appOwner.username !== "spolu") {
```

Comment:
out of consistency even if will be delurked real soon, should be authUser

Reply:
👍